web-react/src/components/Tree/Graph/Edge/AddEdge.js

```diff
@@ -27,9 +115,121 @@ function AddEdge() {
         </div>
       </div>
       <div className="drawer-content">
-        <div>Add Edge fields</div>
+        <div>
+          <form className="form" autoComplete="off">
+            <div className="row">
+              <label>Type</label>
+              <select id="optType" name="type" onChange={update} required>
+                <option value="">Please Select</option>
+                <option value="toQuestion">Question to Question Edge </option>
+                <option value="toSolution">Question to Solution Edge </option>
+              </select>
+            </div>
+            <div className="row">
+              <label>From Question</label>
+              <select id="optType" name="from" onChange={update} required>
+                <option value="">Please Select</option>
+                {
+                
+                  data?.getAllInOne.questions?.map((q, index) => {
+                    return (
+                      <option key={index} value={q.questionId}>
+                        {' '}
+                        {q.questionId}{' '}
+                      </option>
+                    )
+                  })
+                }
+              </select>
+            </div>
+            {state.type === 'toSolution'
+              ? [
+                  <div className="row">
```

Comment:
add key property because you are using array of element for rendering.
If there is single element inside array then don't use array

Reply:
Resolved using () brackets. I tried {..} but it was showing error. Tested it is working also.

Reply:
Correct, you need to use () here.
You cann't directly return any JSX with wrapping in {}.
Mostly {} are use for JavaScript Code.